### DISCUSSION TITLE
[11 Settembre 2023] - Pipeline

### DISCUSSION THREAD

**Discussion by notedo** [CS-Swap/Architettura-degli-Elaboratori#19] [Pipeline]
### Traccia Esercizio

<img width="867" alt="immagine" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/70caac5f-d2bc-486d-93dd-545f5e07aa5d">
<img width="867" alt="immagine" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/7c50a529-ce75-45e3-b07e-dfd906a563ec">
<img width="867" alt="immagine" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/73da12f7-cfe3-4f29-8fe6-0da538dd471e">

**Comment by notedo** [comment]


**Reply by mogliaaaa** [reply]
🟡 5) IF: sub / ID: Beq / EX: Bolla / MEM: Slt / WB: Bolla (Ho messo per errore prima i positivi, ma in realtà il primo intero è negativo)
![image](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/156431154/bddcd918-a2c4-4f7a-b117-23c274c4f792)

**Comment by rimaout** [comment]
<img width="696" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/65cda947-7747-4aa9-af39-aff6e6918e82">
